install/windows.md

```diff
@@ -8,24 +8,35 @@ Wir empfehlen die Installation einer 64-bit Version, falls man ein 64-bit Betrie
 Wenn man sich nicht sicher ist, hat man heutzutage fast immer 64-bit.
```

Comment:
Should we instead just leave the discussion about 32bit and 64bit out of the instructions 🤔 
[mamba releases](https://github.com/conda-forge/miniforge/releases/tag/23.3.1-0) are for 64bit only
and it would be quite unlikely for participants to still have a 32bit system.

And I just found this: On the page for the new method to install the WSL only the buildnumber is mentioned as a
requirement, but on this page [WSL2 Manuall install](https://learn.microsoft.com/en-us/windows/wsl/install-manual)
it is explicitly stated that the system has to be 64bit. 
I guess that settles it. 😅